traffic_ops/testing/api_contract/v4/test_cdns.py

```diff
@@ -0,0 +1,68 @@
+"""Api Contract Test Case for cdns endpoint"""
```

Comment:
file missing license header

Reply:
Added license header to test_cdns.py file